datalake/src/main/java/com/sequenceiq/datalake/entity/DatalakeStatusEnum.java

```diff
@@ -69,6 +69,13 @@ public enum DatalakeStatusEnum {
     DATALAKE_VERTICAL_SCALE_VALIDATION_FAILED(ResourceEvent.DATALAKE_VERTICAL_SCALE_VALIDATION_FAILED),
     DATALAKE_VERTICAL_SCALE_ON_DATALAKE_IN_PROGRESS(ResourceEvent.DATALAKE_VERTICAL_SCALE_ON_DATALAKE_IN_PROGRESS),
     DATALAKE_VERTICAL_SCALE_ON_DATALAKE_FAILED(ResourceEvent.DATALAKE_VERTICAL_SCALE_ON_DATALAKE_FAILED),
+    DATALAKE_HORIZONTAL_SCALE_ON_DATALAKE_VALIDATION_IN_PROGRESS(ResourceEvent.DATALAKE_HORIZONTAL_SCALE_ON_DATALAKE_VALIDATION_IN_PROGRESS),
+    DATALAKE_HORIZONTAL_SCALE_ON_DATALAKE_VALIDATION_FAILED(ResourceEvent.DATALAKE_HORIZONTAL_SCALE_ON_DATALAKE_VALIDATION_FAILED),
+    DATALAKE_HORIZONTAL_SCALE_ON_DATALAKE_IN_PROGRESS(ResourceEvent.DATALAKE_HORIZONTAL_SCALE_IN_PROGRESS),
```

Comment:
why do we have DATALAKE_HORIZONTAL_SCALE_ON_DATALAKE_VALIDATION_FAILED and DATALAKE_HORIZONTAL_SCALE_FAILED states?